table.cc

```diff
@@ -2571,7 +2571,7 @@ future<row_locker::lock_holder> table::do_push_view_replica_updates(const schema
     // We'll return this lock to the caller, which will release it after
     // writing the base-table update.
     future<row_locker::lock_holder> lockf = local_base_lock(base, m.decorated_key(), slice.default_row_ranges(), timeout);
-    return lockf.then([m = std::move(m), slice = std::move(slice), views = std::move(views), base, this, timeout, time_point, source = std::move(source), &io_priority] (row_locker::lock_holder lock) {
+    return lockf.then([m = std::move(m), slice = std::move(slice), views = std::move(views), base, this, timeout, time_point, source = std::move(source), &io_priority] (row_locker::lock_holder lock) mutable {
```

Comment:
This is already part of 77e943e9a3 (I think it shouldn't have been, I missed it in my review...)